src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -43,277 +48,1082 @@ public static void ClassCleanup()
         [TestInitialize]
         public void TestInit()
         {
-            if ("0" != page.GetCalculatorResultText())
-            {
-                page.ClearAll();
-            }
+            page.EnsureCalculatorIsInStandardMode();
+            page.EnsureCalculatorResultTextIsZero();
+            page.CalculatorApp.EnsureCalculatorHasFocus();
```

Comment:
EnsureCalculatorHasFocus should be first, since the other two could fail if the calculator app doesn't have focus.

Reply:
Change will be in my next commit